tool-populate/api/Users.js

```diff
@@ -0,0 +1,110 @@
+var Settings = require('../../js/settings')
+var DemoData = require('../data/DemoData')
+var Parse = require('parse/node')
+var Roles = require('./Roles')
+var useMasterKey = {useMasterKey: true}
+var colors = require('colors')
+var Helpers = require('./Helpers')
+
+function createInitialAdmin(){
+  console.log('\nCreating initial admin user...'.bold);
+  adminUser = new Parse.User()
+  adminUser.set("username", Settings.users[0].user)
+  adminUser.set("password", Settings.users[0].pass)
+  adminUser.set("email", Settings.users[0].user)
+  return adminUser.signUp(null)
+    .then(setAdminACL)
+    .then(function(adminUser){
+      return Roles.getRole('admin')
+    })
+    .then(function(adminRole){
+      adminRole.getUsers().add(adminUser)
+      return adminRole.save(null, useMasterKey)
```

Comment:
When clicking on the Roles -> User relation for `admin`, it correctly displays the admin user. The `role` property on the string.  Line 40 of Users.js we are setting a property 'role' on the users object.  I think the users need to be added to the role relation to be accurate.